style.css

```diff
@@ -79,14 +97,23 @@ button {
 }
 
 #cursor {
-	font-size: 30px;
 	/* Move cursor over the search button left to it */
 	position: relative;
-	left: -50px;
-	bottom: -10px;
+	left: 20px;
+	bottom: -70px;
 	/* For better visibility, like a real cursor */
 	text-shadow: orange 0 0 20px;
 	/* Initially hidden, will be shown when JS plays the search animation */
 	opacity: 0;
-	transition: opacity 0.5s ease;
+	transition: opacity 0.5s ease, bottom 1.5s ease, left 1.5s ease;
+}
+
+#cursor i {
+	/* Increase cursor size */
+	transform: scale(2) !important;
+}
```

Comment:
I also think that this fixes #3 because the cursor isn't colliding/interfering with the text anymore, which was the cause why it was hard to see.